contracts/EscrowFactory.sol

```diff
@@ -48,14 +52,11 @@ contract EscrowFactory is Ownable {
             payeeTrade
         ));
 
-        require(! escrowsCreated[escrowParamsHash], "escrow already exists");
-        
-        EthEscrow escrow = new EthEscrow(
-            address(escrowLibrary)
-        );
+        bytes memory bytecode = abi.encodePacked(ETH_ESCROW_BYTECODE, abi.encode(address(escrowLibrary)));
```

Comment:
What does appending the address of the escrowLibrary to the bytecode do?

Reply:
When the Escrow contract is created its code will initialized with the bytecode and then the rest of the data will be passed to the constructor of the escrow (which takes the escrowLibrary as a parameter).

https://github.com/cwcrypto/arwen-eth-contracts/blob/c872e969ec507b00e8a41b792867c412f2344833/contracts/Escrow.sol#L25